Source/ThirdParty/ANGLE/ANGLE.xcodeproj/project.pbxproj

```diff
@@ -328,8 +328,6 @@
 		31CDFFFD249184B000486F27 /* imageformats.h in Headers */ = {isa = PBXBuildFile; fileRef = 5CCD598B2284FC400018F2D8 /* imageformats.h */; };
 		31CDFFFE249184B000486F27 /* copyimage.h in Headers */ = {isa = PBXBuildFile; fileRef = 5CCD59862284FC400018F2D8 /* copyimage.h */; };
 		31CDFFFF249184B000486F27 /* generatemip.h in Headers */ = {isa = PBXBuildFile; fileRef = 5CCD598C2284FC400018F2D8 /* generatemip.h */; };
-		3A084D472B8C317B00374F23 /* SeparateStructFromFunctionDeclarations.h in Headers */ = {isa = PBXBuildFile; fileRef = 3A084D452B8C317B00374F23 /* SeparateStructFromFunctionDeclarations.h */; };
```

Comment:
These files support `5a4bfd61fd Metal: Separate struct definition from function return`, so It's curious that we're reimporting that change.